1-select-tasks/15.txt

```diff
@@ -1,3 +1,49 @@
 Create a SQL query to return the total number of orders for every month in 1997 year:
 
-| January | February | March | April | May | June | July | August | September | November | December |
\ No newline at end of file
+| January | February | March | April | May | June | July | August | September | November | December |
+
+
+--task15
+SELECT COUNT([o].[OrderID]) AS [January],
```

Comment:
Do this task with CASE, take a look at the example from the stackoverflow:
https://stackoverflow.com/questions/17975229/using-sql-count-in-a-case-statement#answer-17975288